web/test/pages/onboarding/onboarding-shared.test.tsx

```diff
@@ -88,19 +89,22 @@ describe("onboarding - shared", () => {
     expect(screen.getByTestId("step-1")).toBeInTheDocument();
   });
 
-  it("routes to industry page when industry without essential question is set by using industry query string", async () => {
+  it("routes to industry page when industry WITHOUT essential question is set by using industry query string", async () => {
```

Comment:
[pebble] I'm think I might be a little confused, I thought the point of this ticket was that if the industry is set by the query string then we skip onboarding altogether and take the user directly to the dashboard and if they have an essential question then they are taken through onboarding so they can answer it?

Reply:
you're right... I updated the test names incorrectly.
- Without essential question, it goes to dashboard
- With essential question, it stays in onboarding

The test content is right I believe but the test names got mixed up. Good catch!